src/fitnesse/socketservice/SocketService.java

```diff
@@ -71,7 +71,7 @@ private void serviceThread() {
       } catch (SocketException sox) {
         running = false; // do nothing
       } catch (IOException e) {
-        throw new RuntimeException(e);
+        LOG.log(Level.SEVERE, "I/O exception in service thread", e);
```

Comment:
This is the top level in the thread. Propagating at this level would cause the server to stop.